h2o-core/src/main/java/water/rapids/ast/prims/mungers/AstRelevelByFreq.java

```diff
@@ -87,6 +87,13 @@ static int[] getMapping(int[] domainOrder){
         return mapping;
     }
 
+    /**
+     * Take the top N domains and sort rest of the indexes ASC 
+     * @param domainOrder domain order 
+     * @param topN number of top N domains to keep unsorted
```

Comment:
what if topN is higher than domainSize or domainOrder.length? Do we need to care about this input. I would document that the behaviour is undefined at least.